lib/radio_stations.json

```
@@ -48,6 +48,14 @@
     "name": "knrk",
     "description": "anything goes",
     "image": "/home/we/dust/code/pirate-radio/art/infinite.png"
+},
+    {
```

```suggestion
{
```
If you don't mind, this will nudge the `{` over to make it look like the other stations 🙂

---

That looks fine to me, thank you